packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -227,6 +233,45 @@ function CollabSidebarContent( {
 		</div>
 	);
 }
+function CommentPopover( {
+	comment,
+	resultComments,
+	filteredThreads,
+	showCommentBoard,
+	setShowCommentBoard,
+	backgroundColor,
+} ) {
+	const relatedBlock = useMemo( () => {
+		if ( ! comment.id || ! resultComments ) {
+			return null;
+		}
+		return findBlockByCommentId( comment.id, resultComments );
+	}, [ comment.id ] );
+
+	const relatedBlockElement = useBlockElement( relatedBlock );
```

Comment:
I think we can simplify this and avoid passing the whole result `resultComments`. See:

https://github.com/WordPress/gutenberg/blob/fb0953042c86b8786937326e96feb0795d5672b8/packages/editor/src/components/collab-sidebar/comments.js#L114

Reply:
Right! Fixed in https://github.com/WordPress/gutenberg/pull/71856/commits/84fbc0d06a0b796c80c10cdabe7798b337e1178c